src/main/java/seedu/internbuddy/model/company/Company.java

```diff
@@ -45,6 +46,7 @@ public Company(Name name, Phone phone, Email email, Address address, Set<Tag> ta
         this.status = status;
         this.applications = applications;
         this.isFavourite = isFavourite;
+        this.isLong = isLong;
```

Comment:
I think `isLong` is not a very good name for this variable, why not call it `isShowingDetails` for example?